### DISCUSSION TITLE
Add Bzip2?

### DISCUSSION THREAD

**Discussion by WolfJMZ** [101arrowz/fflate#175] [Ideas]
It'd be awesome to have bzip2 functionality incorporated - https://github.com/kirilloid/bzip2-js/blob/master/bzip2.js

I found this works but only has decompress, no compress.

**Comment by 101arrowz** [comment]
This is a bit out of scope for fflate, which focuses on FLATE-based compression methods. I might look into this later and make a library similar to fflate to handle Bzip2 though, and I'll report back here if I find time to do so.
